src/macros.rs

```diff
@@ -185,7 +185,7 @@ macro_rules! valkey_module {
 
         #[no_mangle]
         #[allow(non_snake_case)]
-        pub unsafe extern "C" fn RedisModule_OnLoad(
+        pub unsafe extern "C" fn ValkeyModule_OnLoad(
```

Comment:
We have logic in the Valkey core to allow `RedisModule_` to be used and these are pointed to the equivalent `ValkeyModule_` API.  This allows Modules which use `RedisModule_` APIs are usable on Valkey Servers as well. 

If we change this to `ValkeyModule_`,  it would mean Valkey Modules cannot be loaded on Redis servers.

Just wanted to ask if this is the plan

Reply:
yeah, I thought about it.  You get the message in the logs `Legacy Redis Module`.  I will revert this back for now.